### PR TITLE
Can only be installed in SLE11 platforms.

### PR DESCRIPTION
**Pull request by aplanas**
Depends on crowbar/barclamp-crowbar#1175

### CODE REVIEW

**Comment by tboerger** [comment]
So only SLE 11 nodes can join a pacemaker HA setup?

**Comment by jsuchome** [comment]
I think the only SLE12-supported roles are not possible to put in pacemaker cluster anyway... maybe with the exception of ceph-radosgw...

File: crowbar_framework/app/models/pacemaker_service.rb [hunk 1]
```diff
@@ -31,11 +31,17 @@ def role_constraints
       {
         "pacemaker-cluster-member" => {
           "unique" => false,
-          "count" => 32
+          "count" => 32,
+          "platform" => {
+            "suse" => "/11.*/"
```
Comment (tboerger) [inline crowbar_framework/app/models/pacemaker_service.rb]: What about ```< 12.0```?

File: crowbar_framework/app/models/pacemaker_service.rb [hunk 2]
```diff
@@ -31,11 +31,17 @@ def role_constraints
...
+          }
         },
         "hawk-server" => {
           "unique" => false,
-          "count" => -1
+          "count" => -1,
+          "platform" => {
+            "suse" => "/11.*/"
```
Comment (tboerger) [inline crowbar_framework/app/models/pacemaker_service.rb]: What about ```< 12.0```?

**Comment by aplanas** [comment]
* Express the same constraint using the '<' operator

**Comment by vuntz** [comment]
Technically, it should also be >= 11.3. Not sure if we care about that?

**Comment by aplanas** [comment]
Good point, but is we have SP4 (11.4), using < 12.0 can avoid some updates here.

But I will choose the technically correct one.

**Comment by aplanas** [comment]
Ah sorry. You say >= 11.3, but if we only want that pacemaker can be installed in SLE11 machines, is a <= 11.3, isn't?

I will maintain the < 12.0 here just in case.

**Comment by vuntz** [comment]
@aplanas I really meant SLE11-SP3 and later service packs. Maybe this should `/suse-11\.[3-9]/`

**Comment by aplanas** [comment]
* Restrict to SLE11SP3 or bigger (using @vuntz regexp)

**Pull request merged**